### DISCUSSION TITLE
Unable to connect to any database with MSSQL connector

### DISCUSSION THREAD

**Discussion by unigen-ivlee** [airbytehq/airbyte#44088] [Connector Questions]
Hi There, I'm just starting out evaluating Airbyte OpenSource, I installed on through abctl.  The installation went fine, but I'm trying to add sources.   We have several databases on site from Oracle, MySQL, and MSSQL.  I haven't had any issues connecting Oracle or MySQL  databases, but I'm unable to connect to any MSSQL database that we have.

Connector version is 4.1.8, and System version is 0.63.15

I've tried 3 different databases, SQL Server 2012, SQL Server 2017, and a new Docker SQL Sever 2019.  All of them I'm able to connect through SQL Server Management Studio.  All of them are on prem.

No matter what I've tried, it always takes a while, then errors with a 504.  Does anyone have any ideas, or if I'm missing anything?

The configurations I've tried are below

Host: <in network ipv4 ip address of server>
Port: 1433
Database: <database name> (Tried many, small and large)
Username: <SQL Server Authentication username>, tried several including sysadmin
Password: <password of username>
SSL Method: I've Tried all 3

Update Method: Older ones don't have CDC but I've tried connecting to each
SSH Tunnel: No Tunnel

Optional Fields
Schemas: dbo
JDBC URL Params: <blank>

**Comment by sivankumar86** [comment]
you  can check worker log for more details. Also, attach the worker log here for suggestion.

**Reply by unigen-ivlee** [reply]
Do you think you could point me on where to find the logs?  I tried to view the logs from the documentation below, but both /tmp and /data folders are both empty.  I don't have a connection since I can't setup the source since it won't connect.  Are there other logs I can find for just testing the connection?

https://docs.airbyte.com/operator-guides/browsing-output-logs

**Comment by krutoileshii** [comment]
If you are using MSSQL Instances, you need to find the port number for it. Default 1433 won't work. At least that was the issue for me.

**Reply by unigen-ivlee** [reply]
Thanks for responding.

I just double checked and all our servers are have Dynamic Ports off, and Port is set to 1433.  Also queried sys.dm_exec_connections and everything is showing local_tcp_port as 1433

[a comment by an automated account is not shown]
